cfme/networks/balancer.py

```diff
@@ -0,0 +1,84 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from navmazing import NavigateToSibling, NavigateToAttribute
+from utils.appliance import Navigatable
+from utils.update import Updateable
+from cfme.common import Taggable, SummaryMixin
```

Comment:
**REQUIRED**
Here and below:
`Taggable` and `SummaryMixin` mixins use old nonwidgetastic framework. Please use `WidgetasticTaggable` and remove `SummaryMixin`